torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -367,6 +367,72 @@ def codegen_define(self, kernel: CppTemplateKernel) -> str:
         return result
 
 
+class CppMicroInt8Gemm(CppMicroGemm):
+    DECLARE_KERNEL = r"""
+template <bool accum>
+inline void {{kernel_name}}(
+    const {{input_t}}* __restrict__ A,
+    const {{input2_t}}* __restrict__ B,
```

Comment:
Is extending `CppMicroGemm` to support different dtype for `B` simpler?

Reply:
Thanks for the comments. Another difference is for the data type of `C`. FP32/BF16 Micro GEMM using `output_t` which should be `float32` or `bfloat16` per my understanding, but int8 Micro GEMM using `compute_t` instead of `` which should be `int32`. 

Can we assume same `compute_t` and  `output_t` for FP32/BF16 Micro GEMM and unify to use `compute_t` as  data type of `C`?

Reply:
Thanks for the comment. After a second thought, I think we can extend and reuse most of the code of `CppMicroGemm` instead of introducing `CppMicroInt8Gemm `